config/docker-compose.yml

```
@@ -139,87 +137,79 @@ services:
     volumes:
       - "../certificates:/certificates_to_add"
-      - "./common.yml:/config/common.yml"
-      - "./cards-consultation.yml:/config/application.yml"
-      - ${CONFIG_PATH}:/external-config
+      - "./services/common.yml:/config/common.yml"
+      - "./services/cards-consultation.yml:/config/application.yml"
   web-ui:
     container_name: web-ui
     image: "lfeoperatorfabric/of-web-ui:SNAPSHOT"
     ports:
     - "2002:80"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)